src/temp_workspace/input/sim_config.yaml

```diff
@@ -0,0 +1,37 @@
+  molecular_model:
+    voxel_size: 0.1         # [nm]
+    particle_name: 'toto'   # Name of the particle. Not very important.
+    particle_mrcout: None   # volume map of sample is written.
+
+  specimen_grid_params:
+    hole_diameter: 1200         # [nm]
+    hole_thickness_center: 100  # [nm]
+    hole_thickness_edge: 100    # [nm]
+
+  beam_parameters:
+    voltage: 300           # [kV]
+    energy_spread: 1.3     # [V]
+    electron_dose: 100     # [e/nm**2] dose per image
+    electron_dose_std: 0   # standard deviation of dose per image
+
+  optics_parameters:
+    magnification: 81000        #
+    spherical_aberration: 2.7   # [mm]
+    chromatic_aberration: 2.7   # [mm]
+    aperture_diameter: 50       # [um] in back focal plane
+    focal_length: 3.5           # [mm]
+    aperture_angle: 0.1         # [mrad] of the condenser lens
+    defocus: 1.0                # [um]
+    defocus_syst_error: 0.0     #
+    defocus_nonsyst_error: 0.0  #
+    optics_defocusout: None     # file to write defocus values
+
+  detector_parameters:
+    detector_Nx: 5760           # number of pixels along X axis
+    detector_Ny: 4092           # number of pixels along Y axis
+    detector_pixel_size: 5      # [um]
+    detector_gain: 2            # average number of counts per electron
+    detector_Q_efficiency: 0.5  # detector quantum efficiency
+    noise: 'no'                 # whether quantized electron waves result in noise
+    MTF_params: [0,0,1,0,0]     # to be described. [0,0,1,0,0] is perfect detector.
```

Comment:
@geoffwoollard I think I agree with this:
- if the community uses the star format then we can use the star format

My only concern in moving from yml to star would be: 
- is the star format as easily readable as is the .yml format? 

The advantages with the yml are that:
- everybody knows it (e.g. machine learners who are not necessarily used to cryoEM conventions)
- it is easily readable on GitHub, e.g. in this PR it is easy to see how parameters are set up.

If the star format is easily readable, then we can move to it. Otherwise, I'm not sure...